docs/source/basic_tutorials/non_core_models.md

```diff
@@ -0,0 +1,21 @@
+# Non-core Model Serving
+
+TGI supports various LLM architectures (see full list [here](./supported_models)). If you wish to serve a model that is not one of the supported models, TGI will fallback to transformers implementation of that model. This means you will be unable to use some of the features introduced by TGI, such as tensor-parallel sharding or flash attention. However, you can still get many benefits of TGI, such as continuous batching or streaming outputs.
+
+They can be loaded by:
+
+```python
+from transformers import AutoModelForCausalLM, AutoModelForSeq2SeqLM
+
+AutoModelForCausalLM.from_pretrained(<model>, device_map="auto")``
+
+#or
+
+AutoModelForSeq2SeqLM.from_pretrained(<model>, device_map="auto")
+```
```

Comment:
I'm not sure what's that code is supposed to achieve ? Users are not supposed to run it right ?

I think a mention for `--trust-remote-code` might be nice here.